rust/abacus-base/src/agent.rs

```diff
@@ -132,8 +132,6 @@ pub async fn agent_main<A: BaseAgent>() -> Result<()> {
     crate::oneline_eyre::install()?;
     #[cfg(all(feature = "color_eyre", not(feature = "oneline-errors")))]
     color_eyre::install()?;
-    #[cfg(not(any(feature = "color-eyre", feature = "oneline-eyre")))]
```

Comment:
was this intentionally removed?

Reply:
Yes, sorry, meant to comment on that but apparently I forgot to submit my review. This was removed because I found out it was wrong when running in debug mode. The way it is after this change is correct.